exareme2/controller/api/endpoint.py

```diff
@@ -18,12 +18,25 @@
 )
 from exareme2.controller.node_landscape_aggregator import NodeLandscapeAggregator
 from exareme2.controller.uid_generator import UIDGenerator
+from exareme2.smpc_DTOs import DifferentialPrivacyParams
 
 algorithms = Blueprint("algorithms_endpoint", __name__)
 
+node_landscape_aggregator = None
+cleaner = None
+controller = None
```

Comment:
These variables should probably be in the controller `__init__` file, not here. Would that create any issues?